daemon/daemon.go

```diff
@@ -913,7 +913,8 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine) (*Daemon, error)
 		selinuxSetDisabled()
 	}
 
-	daemonRepo := path.Join(config.Root, "containers")
+	const runDir = "/var/run/docker"
+	daemonRepo := path.Join(runDir, "containers")
```

Comment:
But that's not quite right I think. That data should survive restart.
ping @crosbymichael